### DISCUSSION TITLE
各位大佬，我想问一下如何关闭日志文件的写入

### DISCUSSION THREAD

**Discussion by Arvin-Hugh** [rime/home#1705] [Q&A]
不需要查阅他的日志文件，能否直接关闭所有日志写入？我一天下来，日志文件夹里头好多文件，我也用不到，有个时候由于日志写入，还会造成界面突然卡一下。

**Comment by EoP2** [comment]
没人知道吗，我也想关掉。

**Comment by yjnu** [comment]
1. 日志文件每天清空
2. 写日志文件那点io怎么可能影响使用
3. 正常情况日志并不多，如果日志文件多，那说明配置有问题，一输入就有 warning 产生
